main.go

```diff
@@ -110,6 +110,7 @@ func mainE(ctx context.Context) error {
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CrtFile, "", "Certificate file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.KeyFile, "", "Key file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Service.Prometheus.BaseDomain, "", "Base domain to create Prometheus Ingress resources under.")
+	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Address, "", "Address to access Prometheus UI.")
```

Comment:
What is the difference between address and host?

Reply:
Or `address` and `baseDomain` (which was the original option we had for setting external URL with the original _Ingress_ resource)?

Reply:
Actually, we have both in draughtman CM, address is the same as the host but with https in front :D

Reply:
address has the protocol

```yaml
          Prometheus:
            Address: https://prometheus.g8s.gauss.eu-west-1.aws.gigantic.io
            Host: prometheus.g8s.gauss.eu-west-1.aws.gigantic.io
```